Sources/SnapshotTesting/Snapshotting/SwiftUIView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nathan Mann on 8/17/19.
+//
+
+import Foundation
+import SwiftUI
```

Comment:
This file needs to be wrapped in a compiler directive to retain compatibility with Linux.

Hopefully this does the trick:

``` swift
#if canImport(SwiftUI)
…
#endif
```